slicer/SemiologyVisualisation.py

```diff
@@ -56,7 +55,7 @@
     'Magenta',
 ]
 
-def rename_idx(df: pd.DataFrame, suffix: str):
+def rename_idx(suffix: str):  # df: pd.DataFrame,
```

Comment:
The only thing that needs to be removed is the typing hint. If you remove the `df` parameter, the calls to the function will fail.

Reply:
yikes thanks